streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsMetadataStateTest.java

```diff
@@ -92,8 +92,8 @@ public Object apply(final Object value) {
         });
 
         builder.globalTable("global-topic",
-                            Consumed.with(null, null),
-                            Materialized.<Object, Object, KeyValueStore<Bytes, byte[]>>as(globalTable));
+                Consumed.with(null, null),
```

Comment:
Avoid reformatting in classed that done have any change (please revert those classes).

Reply:
Sure.